test/integration/node/lifecycle_test.go

```diff
@@ -134,7 +134,7 @@ func TestTaintBasedEvictions(t *testing.T) {
 			testCtx = testutils.InitTestScheduler(t, testCtx, true, nil)
 			defer testutils.CleanupTest(t, testCtx)
 			cs := testCtx.ClientSet
-			informers := testCtx.InformerFactory
+			informers := informers.NewSharedInformerFactory(cs, 0)
```

Comment:
don't share the informer factory between scheduler and controller manager, otherwise, event handlers registered in controller manager after the informer factory is started by the scheduler will not take effect. the test passes before because the scheduler uses an independent pod informer with custom ListWatch.

Reply:
Make this a comment in the code.